package/doc/sphinx/source/documentation_pages/analysis/parallelization.rst

```diff
@@ -0,0 +1,360 @@
+.. -*- coding: utf-8 -*-
+
+.. _parallel-analysis:
+   
+=================
+Parallel analysis
+=================
+
+.. versionadded:: 2.8.0
+   Parallelization of analysis classes was added during Google Summer of Code
+   2023 by @marinegor and MDAnalysis GSoC mentors.
+
+Starting with v2.8.0, MDAnalysis, introduces process-based parallelization for
+analysis classes. This section explains the implementation and background for
+parallelization with the :class:`MDAnalysis.analysis.base.AnalysisBase`, what 
+users and developers need to know, when you should use parallelization (almost 
+always!), and when you should abstain from doing so (rarely).
+
+
+How to use parallelization
+==========================
+
+In order to use parallelization in a built-in analysis class ``SomeClass``,
```

Comment:
```suggestion
In order to use parallelization in a built-in analysis class ``SomeAnalysisClass``,
```
Nitpick, feel free to ignore. It would need to be changed elsewhere.